{"uuid": "c8a63037-42e3-4958-a66d-2df49ea6dbd9", "children": ["08857fc0-0184-464d-a402-0a3969230c0a"], "befores": [{"name": "driver", "status": "passed", "start": 1691765161421, "stop": 1691765163070}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\conftest.py\", line 15, in driver\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1691765166368, "stop": 1691765166368}], "start": 1691765161421, "stop": 1691765166368}